packages/suite/src/components/suite/FormattedNftAmount.tsx

```diff
@@ -1,89 +1,127 @@
+                                            tokenValue: token.value,
+                                            collectionName: transfer.name,
+                                        })}
+                                    />
+                                ) : (
+                                    <Row gap={spacings.xxs}>
+                                        <Row>{token.value}x</Row>
+                                        <Translation id="TR_TOKEN_ID" />
+                                    </Row>
+                                )}
+                            </Row>
+                            {isWithLink && network?.networkType === 'ethereum' ? (
+                                <TrezorLink
+                                    href={`${network?.explorer.nft}/${transfer.contract}/${token.id}`}
+                                    color={theme.textSecondaryHighlight}
+                                    variant="underline"
+                                    typographyStyle="label"
+                                >
+                                    <Text maxWidth={145} ellipsisLineCount={1}>
+                                        {token.id}
+                                    </Text>
+                                </TrezorLink>
+                            ) : (
+                                <Text maxWidth={145} ellipsisLineCount={1}>
+                                    {token.id}
+                                </Text>
+                            )}
+                        </Row>
+                    ))}
+                </Column>
+            </HiddenPlaceholder>
+        );
+    }
 
     return (
         <HiddenPlaceholder>
-            <Container className={className}>
+            <Row className={className}>
                 {signValue ? <Sign value={signValue} /> : null}
-                ID:&nbsp;
+                <Box margin={{ right: spacings.xxs }}>
+                    <Translation id="TR_TOKEN_ID" />
+                </Box>
                 {isWithLink ? (
-                    <StyledTrezorLink href={explorerUrl}>
-                        {idComponent}
-                        {symbolComponent}
-                    </StyledTrezorLink>
+                    <TrezorLink
+                        href={
+                            network?.networkType === 'ethereum'
+                                ? `${network?.explorer.nft}/${transfer.contract}/${transfer.amount}`
+                                : undefined
+                        }
+                        color={theme.textSecondaryHighlight}
+                        variant="underline"
+                        typographyStyle="label"
+                        gap={spacings.zero}
+                    >
+                        <Text maxWidth={145} ellipsisLineCount={1}>
+                            {transfer.amount}
+                        </Text>
+                        &nbsp; {symbolComponent}
```

Comment:
does it still need the space between `&nbsp;` `{symbolComponent}`?

Reply:
I think not needed

Reply:
It is needed, without it there will be a space in the underline of the link, as the gap is not considered a normal space and breaks the underline

Reply:
I am talking about 
```
&nbsp;`HERE`{symbolComponent}
```

Reply:
Oh yean, that is not needed indeed